misk/src/main/kotlin/misk/config/HealthChecksModule.kt

```diff
@@ -0,0 +1,16 @@
+package misk.config
+
+import com.google.inject.AbstractModule
+import com.google.inject.multibindings.Multibinder
+import misk.inject.newMultibinder
+
+class HttpHealthChecks(
```

Comment:
This should be decoupled from HTTP exposure. Also, should allow sub-components to register their own healthchecks, rather than the requiring the application install them all manually via this module

Reply:
Sub-components can install their own healthchecks, since they're just adding to the Multibinder. Though this class name is wrong, I'll rename to `HealthChecksModule`.